tests/ExampleApps/DrawPrimitivesApp/minimalradium.cpp

```diff
@@ -477,6 +478,71 @@ void MinimalComponent::initialize() {
                        spline, uint pointCount, const Core::Utils::Color& color, Scalar scale
        = 1.0f
                    );*/
+
+    //// PolyMesh ////
+    {
+        Ra::Core::Geometry::PolyMesh polyMesh;
+        polyMesh.setVertices( {
+            // quad
+            {-1.1_ra, -0_ra, 0_ra},
+            {1.1_ra, -0_ra, 0_ra},
+            {1_ra, 1_ra, 0_ra},
+            {-1_ra, 1_ra, 0_ra},
+            // hepta
+            {2_ra, 2_ra, 0_ra},
+            {2_ra, 3_ra, 0_ra},
+            {0_ra, 4_ra, 0_ra},
+            {-2_ra, 3_ra, 0_ra},
+            {-2_ra, 2_ra, 0_ra},
+            // degen
+            {-1.1_ra, -2_ra, 0_ra},
+            {-0.5_ra, -2_ra, 0_ra},
+            {-0.3_ra, -2_ra, 0_ra},
+            {0.0_ra, -2_ra, 0_ra},
+            {0.0_ra, -2_ra, 0_ra},
+            {0.3_ra, -2_ra, 0_ra},
+            {0.5_ra, -2_ra, 0_ra},
+            {1.1_ra, -2_ra, 0_ra},
+            // degen2
+            {-1_ra, -3_ra, 0_ra},
+            {1_ra, -3_ra, 0_ra},
+
+        } );
+
+        Vector3Array normals;
+        normals.resize( polyMesh.vertices().size() );
+        std::transform(
+            polyMesh.vertices().cbegin(),
+            polyMesh.vertices().cend(),
+            normals.begin(),
+            []( const Vector3& v ) { return ( v + Vector3( 0_ra, 0_ra, 1_ra ) ).normalized(); } );
+        polyMesh.setNormals( normals );
+
+        auto quad = VectorNui( 4 );
+        quad << 0, 1, 2, 3;
+        auto hepta = VectorNui( 7 );
+        hepta << 3, 2, 4, 5, 6, 7, 8;
+        auto degen = VectorNui( 10 );
+        degen << 1, 0, 9, 10, 11, 12, 13, 14, 15, 16;
+        auto degen2 = VectorNui( 10 );
+        degen2 << 14, 13, 12, 11, 10, 9, 17, 18, 16, 15;
```

Comment:
Codacy found an issue: [Found suspicious operator ','](https://app.codacy.com/manual/STORM/Radium-Engine/pullRequest?prid=5831176)

Reply:
@nmellado could you mark these kind of cppcheck/codacy warning as ignored ? (see [here](https://support.codacy.com/hc/en-us/articles/360009180134-Issues-View), but I do not test it)